### PR TITLE
Build Chiselled Ubuntu and .Net containers - WD-3946

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Build Chiselled Ubuntu page and contact form
- Drive-bys on `/silicon`

## QA

- Check out [the demo here](https://ubuntu-com-12995.demos.haus/containers/chiselled/dotnet) and compare it with [the copy doc](https://docs.google.com/document/d/1fUG2SpbfdVDMAdbe7zfuzzvji4oWm-P18D85V2A3F8c/edit) and [the design](https://www.figma.com/file/Jgq60iLkA1su93S0ilHeY2/23.04-U.com---Chiselled-Ubuntu-and-.NET-containers-(WD-3141)?type=design&node-id=70-770&mode=design&t=ui6ZZ6Y6v4ScBZ25-0) 
  - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

- [WD-3946](https://warthogs.atlassian.net/browse/WD-3946)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-3946]: https://warthogs.atlassian.net/browse/WD-3946?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by pedoch** [comment]
The Marketo ID for the form isn't available yet

**Comment by pedoch** [comment]
@juanruitina @camillet16 please take a look when you can. Thanks!

**Comment by juanruitina** [comment]
Nice job, Ochuko. Some minor comments:

- [ ] In the four-column stats block, please reduce the margin between the % figure and the line below.
- [ ] The stats should be within `p` elements, use heading classes instead.
- [ ] Remove the `u-text--muted p-text--small` classes from the disclaimer below the figures. We agreed on the UX phase to keep this rather prominent for accessibility reasons.
- [ ] The last block ("Get in touch") should have a pure white background.
- [ ] The 24/7 text under the fourth icon (the one under "Bug fixing and troubleshooting") is barely readable. @camillet16  can we either make these bigger or change the icon for one without the text?

Thank you!

**Comment by pedoch** [comment]
Hey @juanruitina, I fixed all the issues except the first one, which is a result of the Ubuntu typography as well as the default margins and paddings added by Vanilla. I didn't implement any additional space between them. According to the design guidelines, I can't remove the default margins and padding on texts added by Vanilla because it messes up the baseline grid.

As for the last item (the 24/7 text in the image), I've exported a higher resolution of the image, please let me know if this is fine.

**Comment by juanruitina** [comment]
@camillet16 Can you advise on both the excessive margin below the % and the readability issue with the 24/7 icon?

(I'm afraid the issue with readability has to do with sizing, not so much with resolution)

**Comment by camillet16** [comment]
Screen Shot 2023-07-03 at 6 13 12 PM

- replace with this svg

fix

-------
Screen Shot 2023-07-03 at 5 10 40 PM

- add u-no-margin to both the top and bottom content.

**Comment by pedoch** [comment]
@camillet16 @juanruitina please take a look.

**Comment by pedoch** [comment]
@mtruj013 please take a look when you can. Thanks!

**Comment by camillet16** [comment]
Screen Shot 2023-07-04 at 11 31 11 AM

Add padding-bottom: 1.5rem/24px to paragraph above "Get in touch"

----------

Screen Shot 2023-07-04 at 11 31 46 AM

Add padding-bottom: 1.5rem/24px this section.

**Comment by juanruitina** [comment]
LGTM, nice work!

**Comment by pedoch** [comment]
@camillet16 everything should be good now

**Comment by camillet16** [comment]
LGTM!

File: templates/containers/chiselled/dotnet.html [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+    </div>
+    <p>Data is based on migration from servers to containers <br class="u-hide--small u-hide--medium"/>and from Windows to Ubuntu with chiselled Ubuntu.</p>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-6 col-medium-3">
+      <h2>What is chiselled Ubuntu?</h2>
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:60]:
```suggestion
      <h2>What is chiselled Ubuntu</h2>
```
There's no punctuation on these headings on the doc

File: templates/containers/chiselled/dotnet.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="p-rule" />
+    <div class="col-6 col-medium-3">
+      <h2>Why choose chiselled Ubuntu</h2>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>Chiselled Ubuntu images are made using Canonical’s regular high-quality components and are updated within the same release cycle and with the same support guarantees.</p>
+      <p>There’s an infinite number of slices of the Ubuntu distribution. You can chisel Ubuntu for your exact needs, or rely on our set of pre-built chiselled Ubuntu runtime images.</p>
+      <p>In August 2022, we released the <a href="/blog/install-dotnet-on-ubuntu">first set of pre-built chiselled Ubuntu runtime images</a> for the .NET ecosystem in collaboration with Microsoft.</p>
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:82]: Both of these can be deleted, they're in a different section

File: static/sass/styles.scss [hunk 3]
```diff
@@ -1367,3 +1365,14 @@ hr.p-separator.is-shallow {
 .u-image-separation {
   margin-top: $spv--small;
 }
+
+// utility to add a strong border to the top of a section
+.u-data-border--top {
```
Comment (mtruj013) [inline static/sass/styles.scss:1370]: I don't think this is necessary, there's a [vanilla component](https://vanillaframework.io/docs/patterns/rule#highlighted) that does the same thing

File: templates/containers/chiselled/dotnet.html [hunk 4]
```diff
@@ -0,0 +1,307 @@
...
+        <a href="/containers/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:29]:
```suggestion
      <div class="p-rule--highlight">
```

File: templates/containers/chiselled/dotnet.html [hunk 5]
```diff
@@ -0,0 +1,307 @@
...
+        <p class="p-heading--2 u-no-margin"><strong>20%</strong></p>
+        <p class="p-heading--4 u-no-margin">Better memory usage</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:35]:
```suggestion
      <div class="p-rule--highlight">
```

File: templates/containers/chiselled/dotnet.html [hunk 6]
```diff
@@ -0,0 +1,307 @@
...
+        <p class="p-heading--2 u-no-margin"><strong>40%</strong></p>
+        <p class="p-heading--4 u-no-margin">Better response time</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:41]:
```suggestion
      <div class="p-rule--highlight">
```

File: templates/containers/chiselled/dotnet.html [hunk 7]
```diff
@@ -0,0 +1,307 @@
...
+        <p class="p-heading--2 u-no-margin"><strong>25%</strong></p>
+        <p class="p-heading--4 u-no-margin">Fewer resources</p>
+      </div>
+    </div>
+    <div class="col-3 col-medium-3 p-block">
+      <div class="u-data-border--top">
```
Comment (mtruj013) [inline templates/containers/chiselled/dotnet.html:47]:
```suggestion
      <div class="p-rule--highlight">
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @pedoch ! A few comments in the code:

**Comment by pedoch** [comment]
@mtruj013 I've addressed your comments. Please take a look again, when you can.

**Pull request merged**